referencia/webpayrest/README.md

```diff
@@ -146,57 +126,48 @@ Transacción Completa Mall | `597055555551` Mall <br> `597055555552` Tienda 1 <b
 > ambiente:
 
 ```java
-
+// El SDK apunta por defecto al ambiente de pruebas, no es necesario configurar lo siguiente
+WebpayPlus.Transaction.setCommerceCode(597055555532);
```

Comment:
Shouldn't this be on the first section as well?

Reply:
By first section you mean the section of the previous comment ?